### ISSUE TITLE
libprimme.so does not have a SONAME

### ISSUE THREAD

**Issue by yurivict** [primme/primme#18]
```
Error: /usr/local/lib/python2.7/site-packages/_Primme.so is linked to /usr/local/lib/libprimme.so which does not have a SONAME.  math/primme needs to be fixed.
```

version 2.1

**Comment by yurivict** [comment]
Please also add numbered shared libraries, as they tell if dependencies need to be updated or not.

**Comment by eromero-vlc** [comment]
Fixed in https://github.com/primme/primme/commit/d089bec9382449fdead56acc545cc0f3c54768e5

**Comment by yurivict** [comment]
You added ```-Wl,-soname,libprimme.so.2```, but you also need to install: ```libprimme.so.2```, ```libprimme.so.2.0``` and maybe ```libprimme.so.2.0.0```.

 ```libprimme.so.2.0.0``` should be a file, the rest should be links.

**Comment by eromero-vlc** [comment]
Fixed at b29a50a8b770a277cca63eac6e4ef345653240ce
